donkeycar/templates/complete.py

```diff
@@ -301,7 +301,7 @@ def run(self, img_arr):
 
     # Use the FPV preview, which will show the cropped image output, or the full frame.
     if cfg.USE_FPV:
-        V.add(WebFpv(), inputs=[inf_input], threaded=True)
+        V.add(WebFpv(), inputs=['cam/image_array'], threaded=True)
```

Comment:
I thought that was what I had in my original PR, maybe it got overwritten later.